src/NServiceBus.Core/Encryption/EncryptedValue.cs

```diff
@@ -6,16 +6,28 @@ namespace NServiceBus
     /// Class used to represent an encrypted value with an initialization vector.
     /// </summary>
     [Serializable]
+    [ObsoleteEx(
+        Message = "Message property encryption is released as a dedicated 'NServiceBus.Encryption.MessageProperty' package.",
+        RemoveInVersion = "8",
+        TreatAsErrorFromVersion = "7")]
     public class EncryptedValue
     {
         /// <summary>
         /// The encrypted value represented as a Base64 string.
         /// </summary>
+        [ObsoleteEx(
+            Message = "Message property encryption is released as a dedicated 'NServiceBus.Encryption.MessageProperty' package.",
```

Comment:
in case we want to change the text I'm ok with switching to a constant, otherwise I don't think we gain anything if we agree on the text and now spend more time on switching it over?